create-note/index.js

```diff
@@ -0,0 +1,57 @@
+const functions = require("@google-cloud/functions-framework")
+const axios = require("axios")
+const Tesseract = require("tesseract.js")
+
+const publishMessage = require("./helpers/publish")
+
+functions.http("extractTextHttp", async (req, res) => {
+  await publishMessage("EXTRACT_TEXT_PUBSUB", JSON.stringify(req.body))
```

Comment:
I don't love extracting this TBH, it's only used once and doesn't mean I have to read any less code